css/style.css

```diff
@@ -0,0 +1,87 @@
+/* GLOBAL STYLES */
```

Comment:
loveeeee the global style sheet, as well as page-specific style sheets, to keep up with a universal design but also giving each page that flexibility as well!